lib/core/presentation/pages/collaborator/sub_pages/collaborator_edit_profile_page/widgets/collaborator_edit_expertise_offering_card/widgets/collaborator_edit_expertise_bottomsheet.dart

```diff
@@ -0,0 +1,186 @@
+import 'package:app/core/presentation/widgets/common/button/linear_gradient_button_widget.dart';
+import 'package:app/core/presentation/widgets/lemon_text_field.dart';
+import 'package:app/gen/assets.gen.dart';
+import 'package:app/i18n/i18n.g.dart';
+import 'package:app/theme/color.dart';
+import 'package:app/theme/spacing.dart';
+import 'package:app/theme/typo.dart';
+import 'package:auto_route/auto_route.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_vibrate/flutter_vibrate.dart';
+import 'package:slang/builder/utils/string_extensions.dart';
+
+List<String> expertiseList = [
+  "Artist",
+  "Copy Writer",
+  "Data Analyst",
+  "Designer",
+  "Digital Marketing",
+  "Product Management",
+  "Product Development",
+  "Rapper",
+  "Researcher",
+  "Sales Consultant",
+];
+
+class CollaboratorEditExpertiseBottomSheet extends StatefulWidget {
+  const CollaboratorEditExpertiseBottomSheet({super.key});
+
+  @override
+  CollaboratorEditExpertiseBottomSheetState createState() =>
+      CollaboratorEditExpertiseBottomSheetState();
+}
+
+class CollaboratorEditExpertiseBottomSheetState
+    extends State<CollaboratorEditExpertiseBottomSheet> {
+  List<String> selectedExpertises = [];
+  String searchValue = '';
+
+  @override
+  Widget build(BuildContext context) {
+    final t = Translations.of(context);
+    final colorScheme = Theme.of(context).colorScheme;
+    return Container(
+      padding:
+          EdgeInsets.only(bottom: MediaQuery.of(context).viewInsets.bottom),
+      color: LemonColor.atomicBlack,
+      child: Column(
+        mainAxisSize: MainAxisSize.min,
+        children: [
+          const BottomSheetGrabber(),
+          LemonAppBar(
+            title: t.collaborator.expertise,
+            backgroundColor: LemonColor.atomicBlack,
+          ),
+          Padding(
+            padding: EdgeInsets.only(
+              left: Spacing.smMedium,
+              right: Spacing.smMedium,
+              top: Spacing.xSmall,
+              bottom: Spacing.smMedium,
+            ),
+            child: LemonTextField(
+              hintText: t.common.search.capitalize(),
+              contentPadding: EdgeInsets.all(Spacing.small),
+              onChange: (value) {
+                setState(() {
+                  searchValue = value.toLowerCase();
+                });
+              },
+            ),
+          ),
+          Expanded(
+            child: Padding(
+              padding: EdgeInsets.symmetric(horizontal: Spacing.smMedium),
+              child: ListView.builder(
+                itemCount: expertiseList.length,
+                padding: EdgeInsets.only(bottom: Spacing.large),
+                itemBuilder: (BuildContext context, int index) {
+                  final expertise = expertiseList[index];
+                  final expertiseLower = expertise.toLowerCase();
+                  if (expertiseLower.contains(searchValue)) {
```

Comment:
What if searchValue is empty ?

Reply:
Also don't we handle case where user comeback edit ? should have initial selected item if user already selected before